internal/postgres/application_service.go

```diff
@@ -25,6 +25,8 @@ type dbApplication struct {
 	School               sql.NullString
 	Major                sql.NullString
 	GraduationYear       sql.NullString
+	FirstName			 sql.NullString
```

Comment:
[gofmt](https://golang.org/cmd/gofmt/) your code (also i'd def recommend getting a plugin for your editor that does it automatically)